tracer/test/Datadog.Trace.TestHelpers/LogEntryWatcher.cs

```diff
@@ -22,26 +22,21 @@ public class LogEntryWatcher : IDisposable
     // When the log file gets too big, the logger will roll over and create a new file, but we may still be reading from the old file
     // We must finish reading from the old one before switching to the new one, hence the queue
     private readonly ConcurrentQueue<StreamReader> _readers;
-
+    private readonly DateTime _initialLogFileWriteTime;
```

Comment:
Do we really need this 🤔 I don't think it adds much value, and isn't _actually_ the last write time anyway 🤔